app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx

```diff
@@ -78,6 +78,7 @@ export const BUTTON_STYLES = twv.tv({
     'transition-[opacity,outline-offset,background,border-color] duration-150 ease-in-out',
   ],
   variants: {
+    isCentered: { true: '' },
```

Comment:
What's the purpose of this prop? I'm starting being worry about the amount of boolean props for a button.

Reply:
without `isCentered`, the icon buttons with an outline are slightly off center because of the padding used to center the text. definitely far from an ideal solution, but i just did whatever was the least likely to break all other buttons - if you have a better solution i'd be happy to switch

Reply:
Am I right that this is happening when the button contains only an icon?

Reply:
yup - see here: <https://github.com/enso-org/enso/pull/10256/files/8f11e9e5582d4cd04dba940eaca6addc05ace307#diff-3fe65ff23a2e634db1d9cb4ea510d11e7450db13a58c2b752c7c3cfc3fa8af47R239-R243>

Reply:
oh, i guess one other thing to note is that the compound variant is also used as a hack to get an icon with an exact (16px in this case) pixel size, to avoid icons being more blurry than they need to be. not sure what's a good way to solve that issue

Reply:
Oh, I see, then the problem is with `isIconOnly` prop. Currently, we customize paddings when icon is preset only in `icon` variant. Maybe it's time to extend this behavior?

Reply:
well, i'm not sure the problem is with the `isIconOnly` prop - rather the issue is the `h-[2.5cap] -mt-[0.1cap]` classes that are applied, causing the icons to be (apparently) off center when an outline is applied, and also causing the icons to be not 16px.

as mentioned above, there's probably a better way to solve this, but i picked the option that was least likely to break some button somewhere i might not know about

Reply:
> least likely to break some button somewhere i might not know about

Don't worry, not a huge deal. it's better to break a button visually(a bit though), than introduce a new option(which increase complexity)

Reply:
(to elaborate: iirc it *was* using `isIconOnly`, and i think that was the exact thing that was causing this issue.)

see example usages here for example:
https://github.com/enso-org/enso/pull/10256/files/8f11e9e5582d4cd04dba940eaca6addc05ace307#diff-f599366d794555bea0bd1d9563b890e56eb4a692687fd1ce8e4fcd9a1d22902bR114-R117

without the explicit `isCentered` prop, i think it would fallback to `isIconOnly`

(i do see that there *is* a dedicated `icon` variant, but it doesn't seem to affect the `icon` slot?)